### PR TITLE
SPCTR-3547:  Adding support for 'transform to' for Spectra blocks-2

### PR DESCRIPTION
**Pull request by yash-tingre-10**
### Description
Added _**Transform**_ _**to**_ for `Media & Text` , `Cover`, `Spacer`, `Pagebreak` and `Separator` Blocks

### Screenshots
Sample [Video](https://d.pr/v/F3eoTP) of working of `Media & Text` and `Cover`

Sample [Video](https://d.pr/v/TYGR4j) of working of  `Spacer`, `Pagebreak` and `Separator`

### Types of changes
Changes in block transforms  of Image Block
Support for  `Media & Text` , `Cover`, `Spacer`, `Pagebreak` and `Separator`

### Test Cases
**Note that only first paragraph is converted into caption rest all the paragraphs and inner blocks are ignored**
`Media & Tex`t and `Cover` : https://d.pr/v/F3eoTP  Added font weight and size

`Spacer`, `Pagebreak` and `Separator` : https://d.pr/v/TYGR4j

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/icon-list/block.js [hunk 1]
```diff
@@ -35,4 +35,22 @@ registerBlockType( 'uagb/icon-list', {
 	edit: ( props ) => ( props.attributes.isPreview ? <PreviewImage image="icon-list" /> : <Edit { ...props } /> ),
 	save,
 	deprecated,
+	transforms: {
```
Comment (rahulvarma722) [inline src/blocks/icon-list/block.js:38]: Keep this transform variable in a separate file and import it in this file.

File: src/blocks/separator/block.js [hunk 2]
```diff
@@ -27,4 +28,36 @@ registerBlockType( 'uagb/separator', {
 	attributes,
 	edit: ( props ) => ( props.attributes.isPreview ? <PreviewImage image="separator" /> : <Edit { ...props } /> ),
 	save,
+	transforms: {
```
Comment (rahulvarma722) [inline src/blocks/separator/block.js:31]: Keep this transform variable in a separate file and import it in this file.

File: src/blocks/icon-list/block.js (hunk 1 above)
Comment (yash-tingre-10) [reply src/blocks/icon-list/block.js]: Done with mentioned changes

File: src/blocks/separator/block.js (hunk 2 above)
Comment (yash-tingre-10) [reply src/blocks/separator/block.js]: Done

**Review by rahulvarma722 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -215,6 +215,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 ### x.x.x ###
+* Feature: Transform to - Added transform to option from Core Cover, Media-Text to Spectra Image and Separator, Page Break and Spacer to Spectra Separator.
```
Comment (adi3890) [inline README.md:218]: @yash-tingre-10 as this consists for 2 blocks, lets make it in 2 lines.
I think it will be more readable.

Comment (yash-tingre-10) [reply README.md:218]: Okay Sure

**Pull request merged**